.github/actions/health-checks/action.yml

```diff
@@ -1,59 +0,0 @@
-name: Run Cypress integration tests on a remote environment
-description: Run integration tests on a remote environment
-inputs:
-  deploy-env:
-    description: The environment being deployed (e.g. "prod" or "test")
-    required: true
-runs:
-  using: composite
-  steps:
-    - name: Use Node.js ${{env.NODE_VERSION}}
-      uses: actions/setup-node@v2.1.5
-      with:
-        node-version: 14
-    - name: Cache yarn
-      uses: actions/cache@v2
-      with:
-        path: ~/.cache/yarn
-        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-yarn-
-    - name: Cache Cypress binary
-      uses: actions/cache@v2
-      with:
-        path: ~/.cache/Cypress
-        key: cypress-${{ runner.os }}-cypress-${{ hashFiles('**/package.json') }}
-        restore-keys: |
-          cypress-${{ runner.os }}-cypress-
-    - name: Makefile deploy verification for backend in ${{env.DEPLOY_ENV}} environment
-      shell: bash
-      working-directory: ./ops
-      env:
-          CURL_TIMEOUT: 5
-      run: |
-        echo "::group::Run Makefile verification"
-        make check-${{env.DEPLOY_ENV}}-commit check-${{env.DEPLOY_ENV}}-readiness
```

Comment:
I might have missed this somewhere, but it seems like we purged the call to `make check` from the rest of what we're doing. Did I read that right? If so, was that intentional? The only other place that has it is the `rollback-application` action.

Reply:
Great question! It's not required for the local run. 

I think it would be fine to add back into our cypress workflow that runs against test.simplereport.gov, but I don't think it's required for a couple of reasons. 

- Cypress runs it's own healthcheck against the environment that includes a check for the commit hash.
- The deployTest workflow that calls our cypress workflow runs similar checks called `wait-for-%-slot-release` and `wait-for-%-commit`.

Reply:
@rin-skylight Checking in on whether you think we have readiness check coverage for this or if you think I should add this back in.

Reply:
I would like to see this added back in, personally. I did another check of this, and this is called from our other `deployENV` actions...I missed that initially. I think leaving it in is necessary for the development environments. Otherwise, we may not be alerted to commits not being in the right place for dev testing.

Unless there's something I am missing, of course!

Reply:
I'll add this back in today!

Reply:
So, following back up on this, I'm only seeing the following jobs in the `deployENV` workflows and related actions.

- `make check-${{ env.DEPLOY_ENV }}-readiness`
- `make wait-for-${{ env.DEPLOY_ENV }}-slot-commit`
- `make wait-for-${{ env.DEPLOY_ENV }}-slot-readiness`

I'll go ahead and add it in because I don't see any downside but I'm not seeing this particular make command anywhere else.

Reply:
I added this check into the cypress run!